nifi-nar-bundles/nifi-elasticsearch-5-bundle/pom.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Licensed to the Apache Software Foundation (ASF) under one or more contributor
+license agreements. See the NOTICE file distributed with this work for additional
+information regarding copyright ownership. The ASF licenses this file to
+You under the Apache License, Version 2.0 (the "License"); you may not use
+this file except in compliance with the License. You may obtain a copy of
+the License at http://www.apache.org/licenses/LICENSE-2.0 Unless required
+by applicable law or agreed to in writing, software distributed under the
+License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS
+OF ANY KIND, either express or implied. See the License for the specific
+language governing permissions and limitations under the License. -->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.apache.nifi</groupId>
+        <artifactId>nifi-nar-bundles</artifactId>
+        <version>1.1.0-SNAPSHOT</version>
+    </parent>
+
+    <groupId>org.apache.nifi</groupId>
+    <artifactId>nifi-elasticsearch-5-bundle</artifactId>
+    <packaging>pom</packaging>
+
+    <properties>
+        <lucene.version>6.3.0</lucene.version>
```

Comment:
It has to be in both the processor POM and NAR POM, so when the two bundles were different, I included it in each of the parent bundle POMs. However, I'm consolidating the bundles (per your suggestion a la Kafka), and now it can't be set in the parent POM, now it will have to be set in each NAR/processors pair. This is due to a higher parent POM setting lucene.version for repository stuff, and the ES POM using that property to choose a version of Lucene (for some reason it's not hard-coded in the published POM). Strange but true :)